### PR TITLE
Realization tweaks

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

- Refactors and improves realization engine. It can also realize non-EGO items now.
- Adds ability to realized fragment's EGO. It deals low white damage to enemies in radius and slows them down for a while.
- Adds "passive ability" to big bird's realized EGO. It shows you a message whenever an abnormality escapes.
- Cleans up the code(moved files and icon states to be in the same order everywhere, from ZAYIN to ALEPH).
- Slight buff to Paradise Lost damage(60 --> 70).

## Why It's Good For The Game

- Better code, more functionality and effects.
- A nice ability that can be used as a support tool, as it does not deal too much damage.
- Something to have before I add an active ability. Allows you to always know what is going on.
- Clean.
- PL was balanced around old values and wasn't as useful as before due to it.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm [hunk 1]
```diff
@@ -1,87 +1,68 @@
...
-	var/list/stats = list(FORTITUDE_ATTRIBUTE,
-			PRUDENCE_ATTRIBUTE,
-			TEMPERANCE_ATTRIBUTE,
-			JUSTICE_ATTRIBUTE)
+	var/static/realized_users = list()
+	/// Assoc list 'input = output'
+	var/list/output = list(
+		// ZAYIN
+		/obj/item/clothing/suit/armor/ego_gear/penitence = /obj/item/clothing/suit/armor/ego_gear/realization/confessional,
+		/obj/item/storage/book/bible = /obj/item/clothing/suit/armor/ego_gear/realization/prophet, // TEMPORARY
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm:10]: We could just use the Paradise lost armor?
I mean it's fitting.

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	I come from the end, and I am here to stay for but a moment.\""
 	special = "This weapon has a ranged attack."
 	icon_state = "paradise"
-	force = 60
```
Comment (Kitsunemitsu) [inline code/game/objects/items/ego_weapons/aleph.dm:7]: This does not need a buff tbh. PL has the best healing in the game off an AOE ranged attack.

File: code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm (hunk 1 above)
Comment (EgorDinamit) [reply code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm:10]: Prophet, judging by the sprite, is plague doctor armor, not white night.

File: code/game/objects/items/ego_weapons/aleph.dm (hunk 2 above)
Comment (EgorDinamit) [reply code/game/objects/items/ego_weapons/aleph.dm:7]: It was worse than justitia, whereas it should be the best ALEPH, nearly on par with twilight.

File: code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm [hunk 3]
```diff
@@ -1,87 +1,68 @@
...
-	else
-		to_chat(user, "<span class='userdanger'>This armor is incompatible with the engine.</span>")
+	var/atom/item_out = output[I.type]
+	to_chat(user, "<span class='notice'>The machine is slowly turning [I] into [initial(item_out.name)]...</span>")
+	if(!do_after(user, 5 SECONDS))
+		return
 
+	qdel(I)
+	realized_users |= user.ckey
+	user.adjust_all_attribute_levels(-10)
```
Comment (dsmith328) [inline code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm:64]: Massive

**Review by dsmith328 — APPROVED** [review]
Looks good overall.

**Pull request merged**